PDFTron/pdftron-react-native
# updating react-native

###  Summary
* This branch successfully updates the project example to the latest stable version of react-native as of today 0.64

### Changes 
* Changes are based mostly on rn-diff-purge 
* Some minor changes for the project specifically

### Testing 
* Remove current node_modules folder `rm -rf ./node_modules` in local repo
*  npm install on example 
* clone the repo in `node_modules/react-native-pdftron` just like wiki
* do nom install
* see if you can run the project in iOS and Android
